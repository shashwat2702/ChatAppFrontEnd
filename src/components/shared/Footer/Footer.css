.pageFooter {
    background-color: #f1f1f1;
    font-size: 2rem;
    height: 4rem;
    text-align: center;
    padding-top: 1rem;
    position: absolute;
    bottom: 0;
    width: 100%;
}