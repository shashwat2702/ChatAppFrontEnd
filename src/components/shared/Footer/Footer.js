import React from 'react';
import './Footer.css';

export default function Footer() {
  return (
    <div class="pageFooter">
        This is the footer section
    </div>
  )
}