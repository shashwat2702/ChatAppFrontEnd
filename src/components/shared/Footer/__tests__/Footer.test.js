import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

jest.mock('react-router-dom');

describe('HomePage snapshot',() => {
    it('should render correctly',() => {
        const HeaderSnap = renderer.create(<Footer/>);
        const HeaderJson = HeaderSnap.toJSON();
        expect(HeaderJson).toMatchSnapshot();
    });
});