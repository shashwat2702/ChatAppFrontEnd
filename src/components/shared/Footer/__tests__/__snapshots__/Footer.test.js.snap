// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage snapshot should render correctly 1`] = `
<footer
  className="pageFooter"
>
  This is the footer section
</footer>
`;
